.github/workflows/test.yml

```diff
@@ -80,6 +80,36 @@ jobs:
             configurationFiles: __tests__/stackhawk.yml
             codeScanningAlerts: true
 
+  javaspringvulny-test:
+    name: Java Spring Vulny Test
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create app directory for javaspringvulny
+        run: mkdir -p __tests__/integration-tests/apps
+      - name: Checkout javaspringvulny repository
+        uses: actions/checkout@v2
+        with:
+          repository: kaakaww/javaspringvulny
+          path: ./__tests__/integration-tests/apps/javaspringvulny
+      - name: Run javaspringvulny
+        run: |
+          cd __tests__/integration-tests/apps/javaspringvulny
+          docker-compose up -d
+      - name: Run HawkScan
+        id: run-hawkscan
+        uses: stackhawk/hawkscan-action@main
+        with:
+          apiKey: ${{ secrets.SHAWK_API_KEY }}
+          workspace: ${{ github.workspace }}/__tests__/integration-tests/configs/javaspringvulny/
+          configurationFiles: stackhawk-jsv-json-token.yml
+          sourceURL: https://download.stackhawk.com/hawk/cli
+          version: latest
```

Comment:
These are the defaults - testing them on purpose I presume?

Reply:
I removed these